packages/rum-core/src/domain/longTask/longTaskCollection.spec.ts

```diff
@@ -22,10 +22,12 @@ describe('long task collection', () => {
   })
 
   it('should only listen to long task performance entry', () => {
+    const { notifyPerformanceEntries } = mockPerformanceObserver()
     const { lifeCycle, rawRumEvents } = setupBuilder.build()
+
+    notifyPerformanceEntries([createPerformanceEntry(RumPerformanceEntryType.NAVIGATION)])
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why do we need to notify the `RumPerformanceEntryType.NAVIGATION`? longTaskCollection only listen to lifeCycle.

Reply:
Once the refacto will be completed, longTask will use PerformanceObservable. So ultimately this test will ensure that PerformanceObservable is well configured